### PR TITLE
chore: for devxp, add a quick way to generate a partial embed link

### PR DESCRIPTION
**Pull request by tomkit**
- quick-fix: add a button to generate a partial embed link
- make `returnUrl` required in embed link

Example: `https://api.supaglue.io/oauth/connect?applicationId=a4398523-03a2-42dd-9681-c91e3e2efaf4&customerId=external-customer-hubspot&providerName={{providerName}}&returnUrl={{returnUrl}}`

Screenshot 2023-04-30 at 8 25 20 PM

Screenshot 2023-04-30 at 8 25 22 PM

## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx [hunk 1]
```diff
@@ -67,18 +70,42 @@ export default function Home() {
...
 
+  const handleEmebedLinkClick = async (params: GridRenderCellParams) => {
+    addNotification({ message: 'Copied to clipboard', severity: 'success' });
+
+    // NOTE: assumes one connection per customer
+    // TODO: data-drive `returnURL`
+    await navigator.clipboard.writeText(
+      `https://api.supaglue.io/oauth/connect?applicationId=${applicationId}&customerId=${encodeURIComponent(
+        params.id
+      )}&providerName={{providerName}}&returnUrl={{returnUrl}}`
```
Comment (asdfryan) [inline apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx:81]: Why can't we just use the frontend url (from env var) here?

Comment (tomkit) [reply apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx:81]: For the host?

File: apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx [hunk 2]
```diff
@@ -67,18 +70,42 @@ export default function Home() {
     setMobileOpen(!mobileOpen);
   };
 
+  const handleEmebedLinkClick = async (params: GridRenderCellParams) => {
+    addNotification({ message: 'Copied to clipboard', severity: 'success' });
+
+    // NOTE: assumes one connection per customer
+    // TODO: data-drive `returnURL`
+    await navigator.clipboard.writeText(
+      `https://api.supaglue.io/oauth/connect?applicationId=${applicationId}&customerId=${encodeURIComponent(
```
Comment (asdfryan) [inline apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx:79]: This needs to come from env

File: apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx (hunk 1 above)
Comment (asdfryan) [reply apps/mgmt-ui/src/pages/applications/[applicationId]/index.tsx:81]: Sorry I meant backend url

**Review by asdfryan — APPROVED** [review]

**Pull request merged**